jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -463,14 +447,29 @@ ecma_op_to_string (ecma_value_t value) /**< ecma value */
     return NULL;
   }
 #endif /* ENABLED (JERRY_ES2015) */
-  JERRY_ASSERT (ecma_is_value_boolean (value));
-
-  if (ecma_is_value_true (value))
+  else if (ecma_is_value_true (value))
   {
     return ecma_get_magic_string (LIT_MAGIC_STRING_TRUE);
   }
+  else if (ecma_is_value_false (value))
```

Comment:
Since each condition returns a with a value here and above I'd suggest to use only `if`s instead of `else if`s.